dags/sources/tasks/transform/transform_df.py

```diff
@@ -30,7 +38,7 @@ def merge_duplicates(
     return df_final
 
 
-def merge_produits_accepter(group):
+def _merge_produits_accepter(group):
```

Comment:
je vais faire mon relou, mais j'ai du mal à comprendre ce que fait cette fonction à la lecture de sa signature. 
Elle fait quoi 🤔 ?

Reply:
C'est pour la CMA qui a plusieurs colonnes de produits acceptés quand les autres n'ont qu'une seule colonne avec une liste séparée par des pipes

```suggestion
def _merge_columns_of_accepted_products(group):
```
?

Reply:
aaaah, la _typo_ me donnait l'impression que c'était un verbe d'action, genre "accepter les produits fusionnés" 

La suggestion de modification rend ça vachement plus clair 💯